lib/tasks/version_scheme_counter.rake

```diff
@@ -45,6 +45,8 @@ module VersionSchemeDetection
     end
   }
 
+  TALLIES = { semver: 0, pep440: 0, maven: 0, osgi: 0, calver: 0, unknown: 0, no_versions: 0, total: 0 }
```

Comment:
To be extra safe when working with this constant, should this Hash be frozen?

```suggestion
  TALLIES = { semver: 0, pep440: 0, maven: 0, osgi: 0, calver: 0, unknown: 0, no_versions: 0, total: 0 }.freeze
```

Reply:
That's a good point, may as well.